### PR TITLE
Feature/ub bv 1175

### PR DESCRIPTION
**Pull request by ubozov**
Resolves BV-1175 and BV-1177.

### CODE REVIEW

File: statik/ovpn/templates/server-config.tpl [hunk 1]
```diff
@@ -14,6 +14,8 @@ username-as-common-name
 client-connect "bin/dappvpn{{if .IsWindows}}.exe{{end}} -config config/adapter.config.json"
 client-disconnect "bin/dappvpn{{if .IsWindows}}.exe{{end}} -config config/adapter.config.json"
 script-security 3
+{{if .IsWindows}}#{{end}}up "bin/nat-pf.sh on {{.Server.IP}} {{.Host.Port}}"
+{{if .IsWindows}}#{{end}}down "bin/nat-pf.sh off"
```
Comment (furkhat) [inline statik/ovpn/templates/server-config.tpl]: What about linux?

Comment (ubozov) [reply statik/ovpn/templates/server-config.tpl]: currently only mac/win, after release we will to do implementation on linux

File: statik/ovpn/scripts/nat-pf.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+if [ -n "$1" ]
+then
+	status=$1
+else
+	echo "No arguments supplied (status)."
+	exit 1
+fi
+
+if [ "$status" = "on" ]; then
```
Comment (drew2a) [inline statik/ovpn/scripts/nat-pf.sh]: Please, format this script.

File: statik/ovpn/scripts/nat-pf.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    fi
+
+    # defines default internet interface
+    default=$(/sbin/route get default| grep interface| awk '{print $2}')
+
+    # creates rules
+	rm -f ./config/nat-rules
+
+    nats="nat on $default from $server/24 to any -> ($default)\nnat on $dev from $server/24 to any -> ($dev)"
+	echo $nats >> ./config/nat-rules
```
[the remark this answers is not in the document]
Comment (drew2a) [reply statik/ovpn/scripts/nat-pf.sh]: Please, add double quote

File: statik/ovpn/scripts/nat-pf.sh [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    ports="\npass in proto { tcp, udp } from any to any port $port"
+    echo $ports >> ./config/nat-rules
```
[the remark this answers is not in the document]
Comment (drew2a) [reply statik/ovpn/scripts/nat-pf.sh]: Please, add double quote

[the remark this answers is not in the document]
Comment (ubozov) [reply statik/ovpn/scripts/nat-pf.sh]: updated

File: statik/ovpn/scripts/nat-pf.sh (hunk 2 above)
Comment (ubozov) [reply statik/ovpn/scripts/nat-pf.sh]: updated

File: statik/ovpn/scripts/nat-pf.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (ubozov) [reply statik/ovpn/scripts/nat-pf.sh]: updated

File: statik/ovpn/scripts/nat-pf.sh [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    # defines default internet interface
+    default=$(/sbin/route get default| grep interface| awk '{print $2}')
+
+    # creates rules
+	rm -f ./config/nat-rules
+
+    nats="nat on $default from $server/24 to any -> ($default)\nnat on $dev from $server/24 to any -> ($dev)"
+	echo "$nats" >> ./config/nat-rules
+
+    ports="\npass in proto { tcp, udp } from any to any port $port"
```
Comment (drew2a) [inline statik/ovpn/scripts/nat-pf.sh]: Please, remove redundant spaces/tabs

Comment (ubozov) [reply statik/ovpn/scripts/nat-pf.sh]: updated

**Pull request merged**